developing/eid/README.md

```diff
@@ -6,8 +6,30 @@ We don't care which editor/IDE/debugger that you use - it's your workflow. That
 
 In this section you will find overviews of the range of tools to use and suggestions on how best to configure each tool for the situations we most often use them.
 
-## Contents
+## Editors
 
-List of editors that Truss engineers use:
+List of editors that Truss engineers use (sorted alphabetically):
 
-- TBA
+* Atom
+* Sublime Text
+* vi
```

Comment:
not `vim`?

Reply:
Thanks for pointing that out, both vi and vim. 😆